hw/ip/aes/dv/Makefile

```diff
@@ -38,15 +38,15 @@ BUILD_OPTS += -lcrypto
 # common tests/seqs
 include ${DV_DIR}/../../../dv/tools/common_tests.mk
 
-TEST_NAME       ?= aes_wakeup_test
+TEST_NAME       ?= aes_wakeup
```

Comment:
With the DV regr tool, test names have to match what is provided in the testplan. Either fix the names here or in the testplan...